src/main/java/com/rultor/agents/req/StartsRequest.java

```diff
@@ -173,6 +187,36 @@ public boolean apply(final String input) {
     }
 
     /**
+     * Obtain proxy settings and create a Decrypt instance.
+     * @return Decrypt instance.
+     */
+    private Decrypt decryptor() {
+        final String host = System.getProperty(HOST_KEY, "");
+        Decrypt result;
+        if (host.isEmpty()) {
+            result = new Decrypt(this.profile);
+        } else {
+            final String port = System.getProperty(PORT_KEY, "");
+            int portValue = DEFAULT_PORT;
+            if (!port.isEmpty()) {
+                try {
+                    portValue = Integer.parseInt(port);
+                } catch (final NumberFormatException exception) {
+                    throw new IllegalStateException(
+                        Joiner.on(' ').join(
+                            "Can't parse proxy port",
+                            port
+                        ),
```

Comment:
again, why four lines of code when you may do it in one: `String.format("can't parse proxy port %d", port)`. check this http://www.yegor256.com/2014/06/19/avoid-string-concatenation.html